src/TableView/TableActions.js

```diff
@@ -308,6 +309,16 @@ function deleteError(error) {
   };
 }
 
+function _previousPageIfEmptyAfterDeletion(numberDeleted, state, plural, dispatch) {
```

Comment:
Moreover, I wonder whether this construction is consistent with actions pattern. Definitely it's not, let's say, "a pure action" because it's not connected to store/dispatch (dispatch is passed as an argument). 
Unfortunately, I don't have any idea how to do it better :disappointed: